src/index.html

```diff
@@ -15,8 +15,84 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
You have only two attributes in this tag. There is no need to place each attribute on a new line. It's better to place them in one line.